Eastern-Research-Group/expert-query
# Feature/59 wire up s3 buckets

## Related Issues:
* [EQ-59](https://jira.epa.gov/browse/EQ-59)
* [EQ-70](https://jira.epa.gov/browse/EQ-70)

## Main Changes:
* Wired up a private S3 bucket on the ETL side.
* Removed private S3 environment variables from the app side.
* Added terminology services (glossary) to the ETL process.
  * When running locally this will write the output to the `app/server/app/content-etl` folder. This folder is included in gitignore, because we don't want this in the repo or synced to S3 on build.
  * The `GLOSSARY_AUTH` environment variable was moved from the app side to the etl side.
* Updated the etl process to use configuration in the private S3 bucket.
  * chunkSize - Record limit to pull from services (2000 records at a time).
  * retryLimit - Number of times to retry calling a service (5).
  * retryIntervalSeconds - Amount of time between retries (5 seconds).
  * schemaRetentionDays - Number of days to hold on to old schemas (5 days).

## Steps To Test:
1. Run the ETL process
2. Verify it creates the `app/server/app/content-etl/glossary.json` file
3. Verify the database data is loaded
